TKeazirian.HTTPServer/ResponseBuilder.cs

```diff
@@ -0,0 +1,24 @@
+namespace TKeazirian.HTTPServer;
+
+public class ResponseBuilder
```

Comment:
Have you considered [The Builder Pattern](https://refactoring.guru/design-patterns/builder) ?

Reply:
Actually Travis sent me that but I hadn't sat down to read it yet - will now.

Reply:
[Refactoring Guru](https://refactoring.guru) is a great resource.  As you build out systems, look for things you are trying to accomplish, and you may find there is already a pattern to follow.